cinema/serializers.py

```diff
@@ -1,23 +1,71 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
-class MovieSerializer(serializers.Serializer):
+class ActorSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+    first_name = serializers.CharField(max_length=255)
+    last_name = serializers.CharField(max_length=255)
 
-    def create(self, validated_data):
-        return Movie.objects.create(**validated_data)
+    def create(self, validated_data: dict) -> Actor:
+        return Actor.objects.create(**validated_data)
 
-    def update(self, instance, validated_data):
-        instance.title = validated_data.get("title", instance.title)
-        instance.description = validated_data.get(
-            "description", instance.description
+    def update(self, instance, validated_data: dict) -> Actor:
+        instance.first_name = validated_data.get(
+            "first_name",
+            instance.first_name
+        )
+        instance.last_name = validated_data.get(
+            "last_name",
+            instance.last_name
+        )
+
+        instance.save()
+
+        return instance
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=255)
+
+    def create(self, validated_data: dict) -> Genre:
+        return Genre.objects.create(**validated_data)
+
+    def update(self, instance: Genre, validated_data: dict) -> Genre:
+        instance.name = validated_data.get("name", instance.name)
+
+        instance.save()
+
+        return instance
+
```

Comment:
isn't in better to use ModelSerializer, so that django will do create/update by itself?
Plus you won't need to add these 3 fields and just use Meta class for that